src/routes.tsx

```diff
@@ -1,85 +1,40 @@
 import * as React from 'react';
-import { hashHistory, IndexRoute, Router, Route, RouteConfig } from 'react-router';
+import { hashHistory, Router, RouteConfig } from 'react-router';
 
 import { App } from './App';
 import { Home } from './home';
-import { About } from './about';
-import { Faq } from './faq';
-
-const errorLoading = ( err: any ) => {
-  console.error( 'Dynamic page loading failed', err );
-};
+import { NotFound } from './not-found/NotFound';
 
 const loadModule = ( cb ) => ( componentModule ) => {
   cb( null, componentModule );
 };
 
+const getObjectFirstKey = ( object ) => Object.keys( object )[0];
+
+const removeLeadingSlash = ( routePath ) => routePath.slice( 1 );
```

Comment:
no types, what is routePath? if string, consider `substring` `slice` is more supposed to be used with Array